app/components/page_list_component/view.html.erb

```diff
@@ -31,6 +31,30 @@
 
             </dd>
           </div>
+
+          <% if FeatureService.enabled?(:basic_routing) %>
```

Comment:
```suggestion
          <% if FeatureService.enabled?(:basic_routing) && page.routing_conditions.any? %>
```

Reply:
I was originally trying to think if there would be an issue with older form pages that might not have routing_conditions and if `page.routing_conditions` was `nil` for whatever. Happy for you to ignore the suggestion if you feel theres no chance of this happening and would be pointless.

Reply:
Ah, that makes sense - in this case I'm fairly confident there shouldn't be `nil` values since the relevant API PRs have been merged